### ISSUE TITLE
PLACEHOLDER: Allow admins to invite ChAs to a specific chapter

### ISSUE THREAD

**Issue by shaun-technovation** [Iridescent-CM/technovation-app#4447]

**Comment by dboyer** [comment]
We'll need to add an "organization" field to the admin page for inviting ChAs. This field can be completed two ways:

1. Admin selects an organization from the drop-down list of available orgs. Orgs are added as part of the admin process of creating a chapter.

2. Admin has just created a chapter and clicked the "invite a ChA to this chapter" button as part of that process. The button goes to the ChA invite page with the organization name pre-filled

<img width="723" alt="Screen Shot 2024-02-01 at 12 10 44 PM" src="https://github.com/Iridescent-CM/technovation-app/assets/516764/05253f3a-4002-43dd-a7e6-a35143ae772b">

**Comment by dboyer** [comment]
Completing the second scenario will be done in a separate issue.

**Comment by dboyer** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Technovation-605271851956b1000ec2f24d/issues/Iridescent-CM/technovation-app/4447?planning-poker) with Zenhub @shaun-technovation @viviancan

**Comment by shaun-technovation** [comment]
The basic functionality for this one is on Preview, so an admin can invite a chapter ambassador to a chapter, and then after the chapter ambassador registers they will get associated with that chapter (although I don't think the association is being displayed yet on the platform?)

**Comment by dboyer** [comment]
@shaun-technovation I'm getting an error message when I try to invite a ChA to a chapter. I tried a few different times in different browsers.

https://drive.google.com/file/d/1d49DDL9pJnzsSY3NvOJt3FuJ_GLVHMGg/view?usp=drivesdk

**Comment by shaun-technovation** [comment]
> @shaun-technovation I'm getting an error message when I try to invite a ChA to a chapter. I tried a few different times in different browsers.

This should be fixed now, it was another case of migrations not being run.

**Comment by dboyer** [comment]
Confirming that I was able to invite a chapter ambassador to a specific chapter. The organization did not display on the ChA dashboard or in registration (as expected). 

However, I did not see the ChA listed on the chapter page (/admin/chapters/5#/). Is that a separate thing we need to do? Just wanted to confirm the ambassador name shouldn't be appearing here already.

**Comment by shaun-technovation** [comment]
> However, I did not see the ChA listed on the chapter page (/admin/chapters/5#/). Is that a separate thing we need to do? Just wanted to confirm the ambassador name shouldn't be appearing here already.

Yeah, looks like we haven't build that out yet. I did confirm via the Rails console that the association happened for the last chapter ambassador created though (Spring Time -> STEM for the Future):

```
irb(main):005:0> a = Account.last
=>
#<Account:0x00007fd8ebc81088
...
irb(main):006:0> a.full_name
=> "Spring Time"
irb(main):007:0> a.chapter_ambassador_profile
=>
#<ChapterAmbassadorProfile:0x00007fd8ebbe5638
 id: 131,
 organization_company_name: "Trees",
 job_title: "Flowers",
 account_id: 1612,
 status: "pending",
 created_at: Wed, 28 Feb 2024 13:30:41.310181000 PST -08:00,
 updated_at: Wed, 28 Feb 2024 13:30:41.310181000 PST -08:00,
 bio: "This is my personal summary about trees and flowers and my work as a chapter ambassador. It is really great.",
 intro_summary: nil,
 secondary_regions: [],
 program_name: nil,
 chapter_id: 4>
irb(main):008:0> a.chapter_ambassador_profile.chapter
=>
#<Chapter:0x00007fd8eb98b400
 id: 4,
 name: nil,
 summary: nil,
 organization_name: "STEM for the Future",
 city: nil,
 state_province: nil,
 country: nil>
````
